docs/readme.md

```diff
@@ -163,6 +163,7 @@ Release Notes
     - Analytics: Removed deprecated event names and parameters.
     - Crashlytics (Android): Fixed a bug with missing symbols when enabling
       minification via proguard.
+    - Messaging (iOS): Fixed crash when running on iPhoneOS 12 and older.
```

Comment:
nit: This applies to Firestore as well (and possibly other products).

Reply:
I was waffling on that.  The issue we were seeing with the RC missing stream symbols wasn't present in 8.10.1.  The issue in 8.10.1 seems to do with how messaging hooks into the delegates, but it is possible this did affect other products in some way as well in 8.10.1, so I'll move it to general.